nighthawk/test/test_data/benchmark_http_client_test_envoy.yaml

```diff
@@ -0,0 +1,89 @@
+admin:
```

Comment:
Did you consider using https://github.com/envoyproxy/envoy/blob/master/test/config/utility.h to build this bootstrap? We try to generally avoid hardcoded bootstrap YAML in tests, but if you need gymnastics to make `ConfigHelper` work, this is probably OK.

Reply:
Tbh, I had not seen that, it looks useful. I rewrote some of the bootstrap which allowed a reduction of the amount of yaml needed.. Shall we add improving this further to the track-for-future issue?

Reply:
Yep, sounds like a plane.